specification/resources/resource-manager/Microsoft.Authorization/preview/2023-03-01-preview/privateLinks.json

```diff
@@ -0,0 +1,834 @@
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PrivateLinkAssociation"
+          },
+          "description": "private link association information."
+        }
+      }
+    },
+    "PrivateLinkAssociationPropertiesExpanded": {
+      "type": "object",
+      "properties": {
+        "privateLink": {
+          "type": "string",
+          "description": "The rmpl Resource ID."
+        },
+        "publicNetworkAccess": {
+          "type": "string",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "x-ms-enum": {
+            "name": "publicNetworkAccessOptions",
+            "modelAsString": true
+          }
+        },
+        "tenantID": {
+          "type": "string",
+          "description": "The TenantID."
+        },
+        "scope": {
+          "type": "string",
+          "description": "The scope of the private link association."
+        }
+      },
+      "description": "Private Link Association Properties."
+    },
+    "PrivateLinkAssociation": {
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/PrivateLinkAssociationPropertiesExpanded",
+          "description": "The private link association properties."
+        },
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The plaResourceID."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The operation type."
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The pla name."
+        }
+      }
+    },
+    "ResourceManagementPrivateLink": {
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/ResourceManagementPrivateLinkEndpointConnections"
+        },
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The rmplResourceID."
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The rmpl Name."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The operation type."
+        },
+        "location": {
+          "type": "string",
+          "description": "the region of the rmpl"
+        },
+        "tags": {
+          "description": "The resource tags.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        }
```

Comment:
The id, name, type, location and tags should be defined at top-level, not side properties bag. Also, these fields are already being inherited from proxy resource definition, so no need to define these again here.

Reply:
Not adding those in this PR. They are copied from the existing version 2020-05-01.